bpf/include/bpf/ctx/skb.h

```diff
@@ -4,6 +4,12 @@
 #ifndef __BPF_CTX_SKB_H_
 #define __BPF_CTX_SKB_H_
 
+#ifdef __section_entry
```

Comment:
No need for this, the compiler already throws this including the location where it was defined previously

Reply:
Thank you for the review, I removed the error directive xdp.h, skb.h